xgboost/README.md

```diff
@@ -0,0 +1,212 @@
+ 
+ ```
+ kubectl create -f py-volume.yaml
+ kubectl create -f py-claim.yaml
+ ```
+
+ 
+ * Run docker container on GKE
+   Use the `kubectl` command to run the image on GKE
+   
+   ```
+   kubectl create -f py-pod.yaml
+   ```
+   
+   Once the above command finishes you will have an XGBoost model available at Peristent Volume `/mnt/xgboost/housing.dat`
+
+## Model Export
+The model is exported at location `/tmp/ames/housing.dat`. We will use [Seldon Core](https://github.com/SeldonIO/seldon-core/) to serve the model asset. In order to make the model servable we have created `xgboost/seldon_serve` with the following assets
+
+ * `HousingServe.py`
+ * `housing.dat`
+ * `requirements.txt`
+
+## Model Serving Locally
+We are going to use [seldon-core](https://github.com/SeldonIO/seldon-core/) to serve the model. [HousingServe.py](seldon_serve/HousingServe.py) contains the code to serve the model. Run the following command to create a microservice 
+
+```
+docker run -v $(pwd):/seldon_serve seldonio/core-python-wrapper:0.7 /seldon_serve HousingServe 0.1 gcr.io --base-image=python:3.6 --image-name=${PROJECT_ID}/housingserve
+```
+
+Let's build the seldon-core microservice image. You can find seldon core model wrapping details [here](https://github.com/SeldonIO/seldon-core/blob/master/docs/wrappers/python.md).
+
+```
+cd build
+./build_image.sh
+```
+
+You should see the docker image locally `gcr.io/cloudmlplat/housingserve` which can be run locally to serve the model. Before running the image locally push it to `gcr.io`
+
+```
+gcloud auth configure-docker
+docker push gcr.io/${PROJECT_ID}/housingserve:0.1
+```
+
+Let's run the docker image now
+
+```
+docker run -p 5000:5000 gcr.io/cloudmlplat/housingserve:0.1
+```
+
+Now you are ready to send requests on `localhost:5000`
+
+```
+curl -H "Content-Type: application/x-www-form-urlencoded" -d 'json={"data":{"tensor":{"shape":[1,37],"values":[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37]}}}' http://localhost:5000/predict
+```
+
+```
+{
+  "data": {
+    "names": [
+      "t:0", 
+      "t:1"
+    ], 
+    "tensor": {
+      "shape": [
+        1, 
+        2
+      ], 
+      "values": [
+        97522.359375, 
+        97522.359375
+      ]
+    }
+  }
+}
+```
+
+## Model serving on GKE
+One of the amazing features of Kubernetes is that you can run it anywhere i.e., local, on-prem and cloud. We will show you how to run your code on Google Kubernetes Engine. Start a GKE cluster by following the instructions [here](https://www.kubeflow.org/docs/started/getting-started-gke/). 
+
+Deploy Seldon core to your minikube Kubernetes cluster by following the instructions [here](https://github.com/kubeflow/examples/blob/fb2fb26f710f7c03996f08d81607f5ebf7d5af09/github_issue_summarization/serving_the_model.md#deploy-seldon-core). Once everything is successful you can verify it using `kubectl get pods -n${NAMESPACE}`.
```

Comment:
s/minikube Kubernetes/GKE/
Insert "in the Deploy Seldon Core section" before [here].